doc/source/whatsnew/v0.25.0.rst

```diff
@@ -19,6 +19,36 @@ These are the changes in pandas 0.25.0. See :ref:`release` for a full changelog
 including other versions of pandas.
 
 
+Enhancements
+~~~~~~~~~~~~
+
+.. _whatsnew_0250.enhancements.agg_relabel:
+
+Groupby Aggregation with Relabeling
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Pandas has added special groupby behavior, known as "keyword aggregation", for naming the
+output columns when applying multiple aggregation functions to specific columns (:issue:`18366`).
```

Comment:
I would add a note that the original format is still deprectead (eg. dict-of-dicts), or overkill?